dts/bindings/sensor/kionix,kx022.yaml

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2021 G-Technologies Sdn. Bhd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: KX022 16-Bit digital I2C Accelerometer sensor
+
+compatible: "kionix,kx022"
+
+include: i2c-device.yaml
+properties:
+    int-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INT1 pin
+
+    int-pin-1-polarity:
+      type: int
+      required: false
+      default: 1
+      enum:
+        - 0
+        - 1
+      description: |
+        Polarity of Physical Interrupt Pin 1
+        0: active low
+        1: Active high
+
+    int-pin-1-response:
+      type: int
+      required: false
+      default: 1
+      enum:
+        - 0
+        - 1
+      description: |
+        Response of Physical Interrupt Pin 1
+        0: Cleared by reading INT_REL
+        1: One pulse with a period of 50 us
+
+    full-scale:
+      type: int
+      required: false
+      default: 0
```

Comment:
All defaults must be justified. Please read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings and update the description or remove the use of default here.

This comment applies to every place in this PR where `default` is used in a binding.